stablehlo/transforms/StablehloAggressiveSimplification.cpp

```diff
@@ -1047,6 +1095,7 @@ void populateStablehloCanonicalizationPatterns(MLIRContext *context,
   patterns->add<
       // Arithmetic ops.
       AddOpCanon, SubtractOpCanon, MulOpCanon, CompareOpCanon, SelectOpCanon,
+      SelectIntoMinMax,
```

Comment:
I have considered this naming but was unsure if it accurately reflects the true intent. The simplification actually replaces only the select operation but depends on a combination of two operations. However, this name does seem more natural.

Reply:
I see your point! You are correct that only the "select" operation is replaced. To give a bit more context on my suggestion: because the compare operation ends up with no uses, it will be removed as well, so IMO it is fair to mention both compare and select in the name.